Tribler/Main/Build/Win/setuptribler.py

```diff
@@ -59,12 +59,15 @@
     # (Disabling bundle_files for now -- apparently causes some issues with Win98)
     # options = {"py2exe": {"bundle_files": 1}},
     # zipfile = None,
+    # py2exe includes some dlls which it should not see:
+    # http://stackoverflow.com/questions/20930173/how-to-include-components-of-psutil-in-py2exe-that-py2exe-cant-find - IPHLPAPI.dll was our issue.
+    # http://www.py2exe.org/index.cgi/OverridingCriteraForIncludingDlls
     options={"py2exe": {"packages": packages,
                         "optimize": 2,
                         "skip_archive": True,
                         "dist_dir": os.path.join("dist", "installdir"),
-                        "dll_excludes": ["mswsock.dll", 
-										"MSVCR90.DLL"]
+                        "dll_excludes": ["mswsock.dll", "MSVCR90.dll", "msvcr71.dll",
```

Comment:
@devos50 The `MSVCR90` was an issue of the past. I've looked at what py2exe packs and what it does is packing all MSVC* it can find. the msvcr71 is old (windows XP), but if it present it will also be packed. So I added it there as per the SO answer.